lib/tasks/db.rake

```diff
@@ -0,0 +1,21 @@
+namespace :db do
+  desc 'Erase all tables'
+  task clear: :environment do
+    conn = ActiveRecord::Base.connection
+    tables = conn.tables
+    tables.each do |table|
+      puts "Deleting #{table}"
+      conn.drop_table(table, force: :cascade)
+    end
+  end
+
+  desc 'Task: clear the database, run migrations and seeds'
+  task reseed: [:clear, 'db:migrate', 'db:seed'] {}
+
+  desc 'Task: clear the database, run migrations, seeds and reloads demo data'
+  task :reload do
```

Comment:
@lostie I  `db:migrate:reset` drops the database - which we don't want on remote environments using RDS.